src/components/Integrations/Shopify/Shopify.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import { Helmet } from 'react-helmet';
+import Loading from '../../Loading/Loading';
+import { InjectAppServices } from '../../../services/pure-di';
+
+const Shopify = ({ dependencies: { shopifyClient } }) => {
+  const [shops, setShops] = useState([]);
+  const [isConnected, setIsConnected] = useState(false);
+  const [error, setError] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const getData = async () => {
+      const result = await shopifyClient.getShopifyData();
+      if (result.success && result.value.length) {
+        setIsConnected(true);
+        setShops(result.value);
+      } else if (
+        !result.success &&
+        result.expectedError &&
+        result.expectedError.cannotConnectToAPI
+      ) {
+        setError('Error: No hemos podido conectar con la Api de Shopify, vuelve a intentar luego.');
+      } else if (!result.success) {
+        setError('Error: Error inesperado.');
+      }
+      setIsLoading(false);
+    };
+    getData();
+  }, [shopifyClient]);
+
+  return (
+    <>
+      <Helmet title={'Doppler | Shopify'} />
+      <section style={{ width: '100%', padding: '60px 30px' }}>
+        Panel de control | Integraciones y preferencias avanzadas
```

Comment:
dont use i18n texts?

Reply:
Not for now. This was meant to be very simple.